apps/entry/management/commands/fix_ahhs_data.py

```diff
@@ -0,0 +1,82 @@
+import csv
+import os
+import logging
+from decimal import Decimal
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from apps.entry.models import Figure
+from utils.common import round_half_up
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = "Patch AHHS household size data."
+
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file_path', type=str, help="Path to the CSV file containing the data.")
+
+    # FIXME Use Bulk Manager
+    def patch_figure_household_from_csv(self, file_path):
+        change_in_reported_value_figures = []
+        change_in_unit_figures = []
+        figure_id_not_found = []
+        no_change_in_household_figures = []
+        patched_figure_ids = []
+
+        with open(file_path, 'r') as file:
+            reader = csv.DictReader(file)
+            for row in reader:
+                fig_id = row['figure_id']
+                figure = Figure.objects.filter(id=fig_id).first()
+                if not figure:
+                    figure_id_not_found.append(fig_id)
+                    continue
+
+                if figure.unit != Figure.UNIT.HOUSEHOLD:
+                    change_in_unit_figures.append(figure.id)
+                    continue
+
+                if figure.reported != int(row['figure_reported_value']):
+                    change_in_reported_value_figures.append(figure.id)
+                    continue
+
+                if figure.household_size == row['figure_household_size']:
+                    no_change_in_household_figures.append(figure.id)
+                    continue
+
+                figure.household_size = row.get('figure_household_size')
+                figure.total_figures = round_half_up(
+                    int(row['figure_reported_value']) * Decimal(str(row['figure_household_size']))
+                )
+                figure.save(update_fields=('household_size', 'total_figures',))
+
+                patched_figure_ids.append(figure.id)
+
+        logger.error(f"Figure id not found ids: {figure_id_not_found}")
```

Comment:
Let's use stdout with error for all commands std output
```suggestion
        self.stdout.write(self.style.ERROR(f"Figure id not found ids: {figure_id_not_found}"))
```